README.md

```diff
@@ -1,2 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # capstone-1
 Building an online website for a conference
+
+![screenshot](./images/2021-10-12.png)
+
+.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Application Page url
+
+https://phelian23.github.io/capstone-1/
+
+## Getting Started
+
+> To get a local copy follow the following steps:
+
+- Copy this link https://github.com/phelian23/capstone-1.git
+- Open your terminal or command line
+- Run git clone and paste the link
+- Open the folder with your code editor
+- Create a branch to work on
+
+## Authors
+
+👤 **Phelian23**
+
+- GitHub: [@phelian23](https://github.com/phelian23)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
- [ ] As requested by the previous code reviewer, please do acknowledge the author of the original design.